psydac/fem/tensor.py

```diff
@@ -379,7 +379,7 @@ def spaces( self ):
         return self._spaces
     
     @property
-    def quad_order( self ):
+    defnquads( self ):
```

Comment:
this will not work

Reply:
@saidctb what to use then here?

Reply:
it should be `def nquads`

Reply:
you still didn't fix this issue

Reply:
@mayuri-dhote this one is still not fixed, you fixed another one and you forgot this